<launch>
    <node name="depth_filter_node" pkg="seabot_fusion" type="depth_filter_node" output="screen" ns="fusion" respawn="true" respawn_delay="5">
        <param name="frequency" value="5" />
        <param name="rho" value="1020.0" />
        <param name="g" value="9.81" />

        <param name="filter_window_size" value="6" />
        <param name="filter_mean_half_width" value="2" />

        <param name="filter_mean_width_velocity" value="6" />
        <param name="velocity_dt_sample" value="5" />
        <param name="filter_velocity_mean_half_width" value="2" />

        <param name="velocity_limit" value="0.5" />

        <param name="zero_depth_pressure" value="1.024" />
    </node>

    <node name="temperature_filter_node" pkg="seabot_fusion" type="temperature_filter_node" output="screen" ns="fusion" respawn="true" respawn_delay="5">
      <param name="frequency" value="5" />

      <param name="filter_median_size" value="6" />
      <param name="filter_mean_width" value="2" />
      <param name="filter_mean_width_velocity" value="5" />
      <param name="velocity_delta_size" value="5" />
      <param name="velocity_limit" value="2.0" />
    </node>

	<node name="batteries_filter_node" pkg="seabot_fusion" type="batteries_filter_node" output="screen" ns="fusion" respawn="true" respawn_delay="5">
    <param name="frequency" value="0.2" />
    <param name="filter_median_size" value="6" />
    <param name="filter_mean_width" value="2" />
 </node>

    <node name="internal_sensor_filter_node" pkg="seabot_fusion" type="internal_sensor_filter_node" output="screen" ns="fusion" respawn="true" respawn_delay="5">
        <param name="frequency" value="5" />
        <param name="filter_median_size" value="6" />
        <param name="filter_mean_width" value="2" />
    </node>

</launch>
